src/coreclr/jit/gentree.cpp

```diff
@@ -4986,199 +4986,20 @@ unsigned GenTree::GetScaledIndex()
 }
 
 //------------------------------------------------------------------------
-// gtGetChildPointer: If 'parent' is the parent of this node, return the pointer
-//    to the child node so that it can be modified; otherwise, return nullptr.
+// TryGetUse: Get the def -> def edge for a child of this tree.
```

Comment:
Nit: `def` seems like an odd name choice here, and `def -> def` likewise.

Maybe `child` would be better?

Reply:
Renamed to `operand` as that's what other related APIs use (`ReplaceOperand`, `VisitOperands`, `Operands()`).